config/bot_manager.js

```diff
@@ -66,9 +66,9 @@ module.exports = {
     // sharedSock: {
     //   path: "/tmp/appbuilder.sock"
     // },
-    // tcp: {
-    //   port: "1338",
-    //   accessToken: "li77lePigLi77lePigLe7MeIn"
-    // }
+    tcp: {
```

Comment:
since this is in source control, don't commit this, since we don't want to accidentally install this in a running instance.